app/cluster/integration_test.go

```diff
@@ -103,6 +98,12 @@ func run(m *testing.M) int {
 		return 1
 	}
 
+	defer func() {
+		if err := pool.Purge(resourcePostgres); err != nil {
+			log.Printf("Could not purge resource: %s \n", err)
+		}
+	}()
```

Comment:
How come you're moving this afterwards? Shouldn't we purge it even if we couldn't connect?